<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Detail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="order_num_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Order Number: ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="order_date_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Order Date: ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="shipping_address_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Shipped To: ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="order_items_table"/>
   </item>
   <item>
    <widget class="QLabel" name="promo_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Promo (code) applied!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="grand_total_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Grand Total: ??</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
